modules/ccalib/src/ccalib.cpp

```diff
@@ -445,8 +445,13 @@ bool CustomPattern::findRtRANSAC(InputArray objectPoints, InputArray imagePoints
             OutputArray rvec, OutputArray tvec, bool useExtrinsicGuess, int iterationsCount,
             float reprojectionError, int minInliersCount, OutputArray inliers, int flags)
 {
+    int npoints = imagePoints.getMat().checkVector(2);
+    CV_Assert(npoints != -1);
+    double confidence_factor = minInliersCount / npoints;
```

Comment:
`double confidence_factor = minInliersCount / npoints;`

in divides "int" by "int" and then converts "zero" result to double.
This is why some tests are required...

So,

1. `CV_Assert(npoints > 0);` to avoid other "div by zero" error (it is not recoverable for integer arithmetic, also we don't go deeper with Inf/NaN values).
2. force conversion: `double confidence_factor = (double)minInliersCount / (double)npoints;`

Reply:
Thanks for your review. Fixed now. Should be all good finally.

Reply:
Btw, is there any way that I can take a look at MSVC warnings even when compiling on Linux? Currently even with the warnings as errors option on CMAKE GCC didn't catch the conversion warning.